nixos/modules/services/security/fail2ban.nix

```diff
@@ -202,6 +202,21 @@ in
        '';
       };
 
+      extraSettings = mkOption {
+        type = with types; attrsOf (oneOf [ bool ints.positive str ]);
+        default = {};
+        description = lib.mdDoc ''
+          Extra default configuration for all jails (i.e.: [DEFAULT]). See
+          <https://github.com/fail2ban/fail2ban/blob/master/config/jail.conf> for an overview.
+        '';
+        example = literalExpression ''
+          {
+            bantime = "1h";
```

Comment:
This parameter is already in new PR, it is better to specify another as an example.